docs/canonicalk8s/snap/howto/security/refresh-external-certs.md

```diff
@@ -21,6 +21,13 @@ argument with a YAML-formatted file. For a complete list of available
 certificate keys, see the
 [certificates refresh configuration file reference page][reference page].
 
+```{note} If your cluster uses a mixed certificate management approach where
```

Comment:
Does this mean that if I have external certs and internal certs on my worker node and I just ask it to `sudo k8s refresh-certs --expires-in 10y --timeout 10m` it won't work? I would assume that it would update all the internally managed certs only?

Reply:
Yes, it won't work because there's no way to get the necessary information from the worker's end